index.md

```diff
@@ -21,6 +21,14 @@ title: Distributed SQL query engine for big data
   </div>
 </div>
 
+<div class="trino-birthday-banner">
+  <div class="trino-birthday-content">
+    <h1>Trino is celebrating its 10th Birthday!</h1>
+    <a class="btn trino-birthday-button" href="./blog/2022/08/08/trino-tenth-birthday.html">Click to learn more</a>
```

Comment:
Just use 

Learn more


it should be obvious that it is a button

Reply:
updated!